worker.yml

```diff
@@ -16,6 +16,7 @@ variables:
   gce_project_id: "{{ env `GCE_PROJECT_ID` }}"
   opsmatic_integration_token: "{{ env `OPSMATIC_INTEGRATION_TOKEN` }}"
   papertrail_remote_port: "{{ env `PAPERTRAIL_REMOTE_PORT` }}"
+  travis_cookbooks_url: "{{ env `TRAVIS_COOKBOOKS_URL` }}"
```

Comment:
Looks like this and the line below didn't get ported into the JSON version, causing the build to fail.